### PR TITLE
Update converting-between-strings-and-other-data-types.md

### PR DESCRIPTION
**Pull request by Youssef1313**

### CODE REVIEW

File: docs/visual-basic/programming-guide/language-features/strings/converting-between-strings-and-other-data-types.md [hunk 1]
```diff
@@ -1,25 +1,27 @@
...
-  
-## In This Section  
- [How to: Convert an Array of Bytes into a String in Visual Basic](../../../../visual-basic/programming-guide/language-features/strings/how-to-convert-an-array-of-bytes-into-a-string.md)  
+# Converting between strings and other data types in Visual Basic
+
+This section describes how to convert strings into other data types.
+
+## In This Section
+
+ [How to: Convert an Array of Bytes into a String in Visual Basic](how-to-convert-an-array-of-bytes-into-a-string.md)
```
Comment (mairaw) [inline docs/visual-basic/programming-guide/language-features/strings/converting-between-strings-and-other-data-types.md]:
```suggestion
 [How to: Convert an Array of Bytes into a String in Visual Basic](how-to-convert-an-array-of-bytes-into-a-string.md)  
```

File: docs/visual-basic/programming-guide/language-features/strings/converting-between-strings-and-other-data-types.md [hunk 2]
```diff
@@ -1,25 +1,27 @@
...
-# Converting Between Strings and Other Data Types in Visual Basic
-This section describes how to convert strings into other data types.  
-  
-## In This Section  
- [How to: Convert an Array of Bytes into a String in Visual Basic](../../../../visual-basic/programming-guide/language-features/strings/how-to-convert-an-array-of-bytes-into-a-string.md)  
+# Converting between strings and other data types in Visual Basic
+
+This section describes how to convert strings into other data types.
+
+## In This Section
```
Comment (mairaw) [inline docs/visual-basic/programming-guide/language-features/strings/converting-between-strings-and-other-data-types.md]: this list is actually missing the two last topics from this section. Would you like to add them?

https://docs.microsoft.com/en-us/dotnet/visual-basic/programming-guide/language-features/strings/converting-between-strings-and-other-data-types

image

**Review by mairaw — CHANGES_REQUESTED** [review]
Thanks @Youssef1313. Left a few suggestions.

File: docs/visual-basic/programming-guide/language-features/strings/converting-between-strings-and-other-data-types.md (hunk 2 above)
Comment (Youssef1313) [reply docs/visual-basic/programming-guide/language-features/strings/converting-between-strings-and-other-data-types.md]: @mairaw, Sure. I'll add them.

Comment (Youssef1313) [reply docs/visual-basic/programming-guide/language-features/strings/converting-between-strings-and-other-data-types.md]: @mairaw, I've added them. I noticed that some file names contains small words like `a`, `to`, .. I believe this shouldn't be like that. Am I correct ?

Comment (mairaw) [reply docs/visual-basic/programming-guide/language-features/strings/converting-between-strings-and-other-data-types.md]: Yeah, but when we migrated content from MSDN, the migration process wasn't aware of these rules. But not sure if it's worth changing now but we do pay attention when creating new articles or moving them.

**Review by mairaw — APPROVED** [review]
Thanks for adding the missing articles!

**Pull request merged**